docs/administration/configuration/configuring-access/index.md

```diff
@@ -0,0 +1,6 @@
+# Configuring user access
+
+By default confirmation of a user’s email address on sign up is enabled.
+The user is unable to sign in until they confirm their email address.
+
+Also, it's possible to hide the sign up form in web UI by using `hide_signup` helm value.
```

Comment:
```suggestion
Also, it's possible to hide the sign up form in web UI by using `hide_signup` Helm value.
```

What do you think about linking "`hide_signup` Helm value" to https://github.com/reanahub/reana/tree/master/helm/reana so the user has better reference?